src/status_im2/contexts/activity_center/events.cljs

```diff
@@ -119,10 +119,14 @@
                      accepted
                      (not dismissed))
                 (toasts/upsert cofx
-                               {:icon       :placeholder
-                                :icon-color colors/primary-50-opa-40
-                                :title      (i18n/label :t/contact-request-accepted-toast
-                                                        {:name (:alias message)})})
+                               {:user  {:full-name         name
+                                        :profile-picture   (:identicon message)
```

Comment:
A non-blocking comment:

We are passing identicon as a profile picture to the `user-avatar` not the actual profile picture of the user (if he/she has any).  

I know this requires an update in the `user-avatar` component to receive the public key of any user, and then get the profile picture from there (via re-frame subscription). I know this is not in the scope of this PR, We can handle this separate issue.

In meantime, A suggestion to prevent the ugly UI, can we skip the identicon and set the initials as the default profile picture. At least, this will go well with the design. WDYT?

Reply:
@smohamedjavid as far as I understood the intention is (at least in some issues it is stated so, like here https://github.com/status-im/status-mobile/issues/14622) that we keep unfinished things ugly on purpose, to make them visible. The thing which is different but looks good doesn't pass that requirement :) 

@churik @pavloburykh wdyt?

Reply:
I agree 😄 👍